sklearn/datasets/lfw.py

```diff
@@ -44,14 +44,15 @@
 logger = logging.getLogger(__name__)
 
 
-BASE_URL = "http://vis-www.cs.umass.edu/lfw/"
```

Comment:
Hmm. Can we assume no backwards compatibility issues with these names? I'm fine with this.